### DISCUSSION TITLE
How to change GitLens GitLab/GitHub access token before it's expired?

### DISCUSSION THREAD

**Discussion by ig0rmin** [gitkraken/vscode-gitlens#3368] [Q&A] [answered]
When you first install GitLens and not yet connected it to GitHub/GitLab it offers to enter access token in the hover window. But how to change the access token after it was configured? I could not find it anywhere in the settings nor in the command palette.

**Comment by jkelroy** [comment, marked as answer]
Hello @ig0rmin, in Source Control (by default), you can navigate to the Remotes sidebar, hover over a remote, and select Disconnect Remote Integration. After that, you can reconnect and then you will be prompted to provide a new access token.

![Screenshot 2024-07-16 at 1 58 06 PM](https://github.com/user-attachments/assets/b56fa75d-9a75-4f5f-a771-f4abbdc0ae60)
